admin/tabs/general/alertBanner/AlertBannerModel.ts

```diff
@@ -104,6 +110,50 @@ export class AlertBannerModel extends HoistModel {
         this.refreshAsync();
     }
 
+    loadPreset(preset: PlainObject) {
+        this.formModel.setValues({...preset, expires: null});
+    }
+
+    @action
+    addPreset() {
+        const {message, intent, iconName, enableClose} = this.formModel.values;
+        const dateCreated = fmtDateTime(Date.now());
+        this.savedPresets = sortBy(
+            [...this.savedPresets, {message, intent, iconName, enableClose, dateCreated}],
+            [preset => preset.intent, preset => preset.message]
```

Comment:
Believe you can use shortcut form of passing the field names as strings instead of arrow functions - i.e. second arg can be `['intent', 'message']` - this is common across many lodash utils